pkgs/tools/misc/xprite-editor/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, gtk3
+, AppKit
+, pkg-config
+, python3
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "xprite-editor-unstable";
+  version = "2019-09-22";
+
+  src = fetchFromGitHub {
+    owner = "rickyhan";
+    repo = "xprite-editor";
+    rev = "7f899dff982642927024540e4bafd74e4ea5e52a";
+    sha256 = "1yp9zagli69rs0qwbwlk7fk0f9hzir69vh8xrlzp1jb65mmbmxm4";
```

Comment:
it produces a different hash on Linux and darwin because the repository has a [unicode fiilename](https://github.com/rickyhan/xprite-editor/blob/7f899dff982642927024540e4bafd74e4ea5e52a/config/palettes/Sweet%20Guaraná.hex). I'm going to fix it later.